Makefile

```diff
@@ -266,9 +266,8 @@ test-unit: clean-cov generate fmt vet ## Run Unit tests.
 namespace: ## Creates a namespace where to deploy Kuadrant Operator
 	kubectl create namespace $(KUADRANT_NAMESPACE)
 
-.PHONY: local-setup
-local-setup: $(KIND) ## Deploy locally kuadrant operator from the current code
-	$(MAKE) local-env-setup
+.PHONY: local-deploy
+local-deploy: ## Deploy Kuadrant Operator in the cluster pointed by KUBECONFIG
```

Comment:
This is not part of the fix, but an easy way to separate the targets and use them for testing other ways of setup